code/game/objects/structures/tank_holder.dm

```diff
@@ -0,0 +1,134 @@
+/obj/structure/tank_holder
+	name = "tank holder"
+	desc = "A metallic frame that can hold tanks and extinguishers."
+	icon = 'icons/obj/tank.dmi'
+	icon_state = "holder"
+
+	custom_materials = list(/datum/material/iron = 2000)
+
+	density = FALSE
+	anchored = FALSE
+	pass_flags = LETPASSTHROW
+	max_integrity = 20
+
+	resistance_flags = FIRE_PROOF
+
+	var/obj/item/tank = null
```

Comment:
remove `= null`. it's not needed. also, even though it's a little self-explanatory, mind putting a comment with ///?